app/helpers/registrations_helper.rb

```diff
@@ -32,7 +47,7 @@ def handle_user_creation(created_user)
     else
       created_user.delete
```

Comment:
Why are we deleting the user here?

Reply:
In the CSV building, if a valid user is saved but has errors, it ends up clogging the db so I clear em